src/frontend/src/common/components/library/data_subview/components/MoreActionMenu/index.tsx

```diff
@@ -66,9 +69,15 @@ const MoreActionsMenu = ({
         onClose={handleClose}
         getContentAnchorEl={null}
       >
+        {usesFeatureFlag(FEATURE_FLAGS.editSamples) && (
+          <MenuItem onClick={() => undefined}>
```

Comment:
Nit: We have a `noop` function in `src/frontend/src/common/constants/empty.ts` -- I like using it for stuff like this just because I think it very clearly describes what it's doing to any dev reading it.

Reply:
oh cool didn't realize we had that thanks!